index.html

```diff
@@ -569,7 +572,7 @@ <h6 class="my-3">Quality Product</h6>
               </p>
             </div>
           </div>
-          <div class="col-3">
+          <div class="col-md-3">
             <div class="layer-2 text-center">
               <img src="img/24-hours-support 1.png" alt="" />
               <h6 class="my-3">24/7 Support</h6>
```

Comment:
"Trending Products" Card issue same as the latest products

Reply:
Done